apps/entity-service/src/entities/dto/entity-query.dto.ts

```diff
@@ -26,6 +26,10 @@ export class EntityQueryDto extends IntersectionType(QuerySort, NumberPage) {
   @IsOptional()
   search?: string;
 
+  @ApiProperty({ required: false })
+  @IsOptional()
+  q?: string;
```

Comment:
What is this? Could definitely use a better name, and if that's not possible, for sure needs a description.

Reply:
Hmm, I'm still not understanding why we need a second search query. But if it really is necessary, can we at least give it a better name? `filter` or `searchFilter` come to mind.